_data/lang/it/footer-1.yml

```diff
@@ -0,0 +1,50 @@
+- title: Risorse
+  subfolderitems:
+  - name: Riguardo Monero

```

Comment:
'Riguardo a Monero' sarebbe la forma corretta, 'Riguardo Monero' viene comunque grammaticalmente accettata, si potrebbe forse considerare qualcosa come 'Informazioni su Monero'.

Reply:
ps: speak english, if you can! :)

Reply:
Sorry, I was thinking that Italian was the language for the Italian translators.
My mistake, excuse me. I try to fix it.

Reply:
'Riguardo a Monero' would be the correct form, 'Riguardo Monero' is still grammatically accepted, maybe we should consider something like 'Informazioni su Monero'.

Reply:
Yes, the italian language is the language for the Italian translators. But you should remember that not everyone here is italian! :)

Reply:
I repeat that was my mistake, excuse me. 
Anyway, I fixed it.

Reply:
Changed into 'Informazioni su Monero'